### PR TITLE
Cypress Test For add user to organization

### PR DESCRIPTION
**Pull request by Vayras**
## Describe your changes
Added cypress test for adding a user to org

## Issue ticket number and link
issue: #404 

## Type of change
Cypress Test 

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [X] I have tested on Cypress locally on Chrome and Firefox

### CODE REVIEW

**Comment by Vayras** [comment]
Cypress Test passes on local machine, failed here somehow

File: cypress/e2e/17_create_org_user.cy.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      cy.contains('Add User').click();
+      cy.wait(1000);
+    });
+    cy.wait(1000);
+
+    cy.get('#sphinx-top-level-overlay').within(() => {
+      cy.contains('Assign').click();
+      cy.wait(1000);
+    });
+
```
Comment (elraphty) [inline cypress/e2e/17_create_org_user.cy.ts:41]: @Vayras please click on the settings icon and confirm that the roles you assigned persisted, thanks.

Comment (Vayras) [reply cypress/e2e/17_create_org_user.cy.ts:41]: Sure will do

**Comment by Vayras** [comment]
@elraphty made the necessary changes. Thanks

**Pull request merged**